patches/chrome-installer-util-google_update_settings.cc.patch

```diff
@@ -0,0 +1,61 @@
+diff --git a/chrome/installer/util/google_update_settings.cc b/chrome/installer/util/google_update_settings.cc
+index 42a2c0cc1b9576aa5f6c03e615e4b8450611831d..5142fdf31a4212b79a0d9d2ed99908b7c0e6820c 100644
+--- a/chrome/installer/util/google_update_settings.cc
++++ b/chrome/installer/util/google_update_settings.cc
+@@ -36,7 +36,11 @@ using base::win::RegKey;
+ using installer::InstallationState;
+ 
+ const wchar_t GoogleUpdateSettings::kPoliciesKey[] =
++#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
++    L"SOFTWARE\\Policies\\BraveSoftware\\Update";
++#else
+     L"SOFTWARE\\Policies\\Google\\Update";
++#endif
+ const wchar_t GoogleUpdateSettings::kUpdatePolicyValue[] = L"UpdateDefault";
+ const wchar_t GoogleUpdateSettings::kDownloadPreferencePolicyValue[] =
+     L"DownloadPreference";
+@@ -50,7 +54,7 @@ const int GoogleUpdateSettings::kCheckPeriodOverrideMinutesMax =
+ 
+ const GoogleUpdateSettings::UpdatePolicy
+ GoogleUpdateSettings::kDefaultUpdatePolicy =
+-#if defined(GOOGLE_CHROME_BUILD)
```

Comment:
are we doing this for all `GOOGLE_CHROME_BUILD` guards in this file? If so it would be easier to just define GOOGLE_CHROME_BUILD if BRAVE_CHROMIUM_BUILD && OFFICIAL_BUILD and then undef at the end

Reply:
isn't the reverse just as likely?

Reply:
done with `#define GOOGLE_CHROME_BUILD`